rosdep/base.yaml

```diff
@@ -4626,6 +4626,321 @@ libpcl-all-dev:
   rhel: [pcl-devel]
   slackware: [pcl]
   ubuntu: [libpcl-dev]
+libpcl-apps:
```

Comment:
You could add `alpine: [pcl-libs]` as well, and also the packages on homebrew like here: https://github.com/ros/rosdistro/pull/35137/files#diff-86e444436e8b275218b0f9fce40452b3a5a24dad5f9bc554b8ec9737134a3684